spec/features/bus_delete_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe 'deleting a bus' do
+  context 'with admin privilege' do
+    let!(:bus) { create :bus }
```

Comment:
This `let` statement can go after the `describe` so that you state it once rather than three times.